### ISSUE TITLE
Not possible to create order

### ISSUE THREAD

**Issue by grillshopen** [PayEx/PayEx.Magento#66]
Hi,
when switching to live from testing with all the correct credentials it's not possible to create order with invoice or with part payment.
The error i get is validation error. I have filled in my SSN and fetched my address, that is all working fine but when I try to complete the order i receive a error message that there's some kind of validation error.
Card payments are working 100% fine.

here is the logs:
`2017-12-01T10:17:20+00:00 DEBUG (7): Action: Validate
2017-12-01T10:17:21+00:00 DEBUG (7): PxOrder.GetAddressByPaymentMethod:Process online request: Method [GetAddressByPaymentMethod] ProcessId [2130766] Status[received]
2017-12-01T10:17:21+00:00 DEBUG (7): PayEx: PxOrder.GetAddressByPaymentMethod ValidationError_InvalidParameter (Process online request: Method [GetAddressByPaymentMethod] ProcessId [2130766] Status[received])
2017-12-01T10:17:37+00:00 DEBUG (7): Action: Validate
2017-12-01T10:17:45+00:00 DEBUG (7): Action: Validate
2017-12-01T10:17:45+00:00 DEBUG (7): Action: Validate
2017-12-01T10:17:46+00:00 DEBUG (7): Action: Validate
2017-12-01T10:17:46+00:00 DEBUG (7): Action: Order
2017-12-01T10:17:46+00:00 DEBUG (7): Action: getOrderPlaceRedirectUrl
2017-12-01T10:17:46+00:00 DEBUG (7): Controller: redirect
2017-12-01T10:17:46+00:00 DEBUG (7): PxOrder.Initialize8:OK
2017-12-01T10:17:46+00:00 DEBUG (7): PayEx: PxOrder.AddSingleOrderLine2 OK (OK)
2017-12-01T10:17:47+00:00 DEBUG (7): PayEx: PxOrder.AddSingleOrderLine2 OK (OK)
2017-12-01T10:17:47+00:00 DEBUG (7): PayEx: PxOrder.AddOrderAddress2 OK (OK)`

This is very frustrating because we've had this error since day 1 (about 2 months now) and we still can't use invoice or part payments

**Comment by grillshopen** [comment]
Hi. Is there any progress on this issue?

**Comment by olegisk** [comment]
Hello, @grillshopen 
Please try to disable "Perform validation of SSN before checkout" option in admin backend

**Comment by grillshopen** [comment]
Hi.
I’ve tried tried that and pretty much everything else that can be done through admin for the payex extension. It is 100% for sure something with the code.

**Comment by aait** [comment]
Hi,

This feels like a problem with the production setup of your account. I have contacted PayEx and asked them to recheck their logs. In any case, if you can provide us with credentials for file access it would be helpful. Please contact us at info@aait.se

**Comment by olegisk** [comment]
Hi @grillshopen,
Please try module from this branch https://github.com/PayEx/PayEx.Magento/tree/feature/bugfixes

**Comment by olegisk** [comment]
See https://github.com/PayEx/PayEx.Magento/commit/c1f8dd083bf6acdbf27571aa26c707e99610f07e

**Comment by grillshopen** [comment]
ok, so. I've updated your extension and I'm not receiving a validation error anymore and I can pass through SSN on billing, which is good. But I still can't create an order. the error is that it says that the client doesn't have a valid credit so it's not possible to purchase with invoice or part payment.
![image](https://user-images.githubusercontent.com/34162450/33717676-2e9d0cb2-db5b-11e7-99dd-155a5d5246fd.png)

I've tried with 2 different SSNs and it didn't work with any of those two. One is my personal SSN and the other is my collegues. It's an order on only 100 SEK.
